### DISCUSSION TITLE
Push PDK layers/layerviews to klayout live

### DISCUSSION THREAD

**Discussion by raimund89** [gdsfactory/gdsfactory#2497] [Q&A]
Hi all,
I've tried to find an answer to this question in the documentation, but couldn't find any information on how to approach this. So hopefully you can help me out :)

I'm using the klayout integration to live update layouts generated by gdsfactory. I've generated and activated my own PDK defining a set of layers and corresponding layerviews (base_pdk=generic_pdk). However, to get the correct layer names, colors and patterns, I need to separately export the layerviews to a .lyp file and open that in klayout. 

Is there a way to 'push' the PDK layerviews to klayout when the PDK is activated? That would make sure that the names and colors are always up-to-date with whatever PDK is used in the Python scripts..

Thanks in advance!

**Comment by joamatab** [comment]
What i do is to have a install_tech that creates a symbolic link
https://github.com/gdsfactory/ubc/blob/main/install_tech.py

this links your LYP to Klayout home folder LYP

then use another script to convert your YAML into LYP
https://github.com/gdsfactory/ubc/blob/main/ubcpdk/tech.py
